networking/hardware_networks/about-sriov.adoc

```diff
@@ -19,49 +20,22 @@ SR-IOV Network Operator. The SR-IOV Network Operator creates and manages the
 components of the SR-IOV stack. The Operator provisions the following
 components:
 
-* Provision the SR-IOV network operator deployment on master nodes.
```

Comment:
It might be good to keep a simple list of items managed by SR-IOV Operator for an overview of what's deployed in the openshift-sriov-network-operator namespace.

Reply:
@zshi-redhat, I'll try updating with a definition list instead; It's easily scannable, similar to a list.

Reply:
@zshi-redhat, updated with one possible approach to this.

Reply:
lgtm